src/main/java/ch/njol/skript/classes/data/DefaultComparators.java

```diff
@@ -464,18 +464,21 @@ public boolean supportsOrdering() {
 		ItemType lava = Aliases.javaItemType("lava");
 		Comparators.registerComparator(DamageCause.class, ItemType.class, new Comparator<DamageCause, ItemType>() {
 			@Override
-			public Relation compare(final DamageCause dc, final ItemType t) {
+			public Relation compare(DamageCause dc, ItemType t) {
```

Comment:
Are these supposed to be non-final? You aren't changing them, so it seems strange that you got rid of the `final`s here

Reply:
Skript's code conventions say not to use the final modifier on local variables and parameters: 
![image](https://user-images.githubusercontent.com/29547183/126289160-752ab63b-ec7f-4798-aad4-ad9fa0ceb9ae.png)

A lot of Skript's code is from before these code conventions existed, so whenever I modify some code in a PR, I also reformat the code if needed

Reply:
Ah, that makes sense.